store/index.js

```diff
@@ -53,15 +53,14 @@ export const getters = {
 export const actions = {
 
     async getUnitList(store, params) {
-        return await axios.get(api_url + '/units', params).then(({data}) => {
+        return await axios.get(api_url + '/units?seasonId=' + params.seasonId).then(({data}) => {
             return data;
         }).catch((err) => {
             console.log(err)
         });
     },
     async getSeasonList(store, params) {
-
-        return await axios.get(api_url + '/seasons', params).then(({data}) => {
+        return await axios.get(api_url + '/seasons').then(({data}) => {
```

Comment:
Codacy has a fix for the issue: [Replace `.get(api_url·+·'/seasons').then(({data` with `⏎············.get(api_url·+·"/seasons")⏎············.then(({·data·`](https://app.codacy.com/gh/Eight-Corner/front_toche_web/pullRequest?prid=9690065)
```suggestion
        return await axios
```